### DISCUSSION TITLE
Date Filter for Number Cards

### DISCUSSION THREAD

**Discussion by bibinqcs** [frappe/insights#252] [Q&A] [answered]
Hi Team,

I am trying to create an insights dashboard for our client and are facing an issue with Number Cards, I am not able to use Date Filters. 
Wanted to if this is possible or is it an bug that it is not working for us

Using Insights V2.1.4

**Comment by nextchamp-saqib** [comment, marked as answer]
You should be able to use date filters with number cards.

Does the source tables of the query associated with your number card have a date field?

**Reply by bibinqcs** [reply]
We were trying with SQL Editor that is why it was not working
